scoring-periods/0.md

```diff
@@ -0,0 +1,16 @@
+## `Scoring Period 0`
+### _DD/MM/YY - DD/MM/YY_
+<br>
+#### Announcement
+Alongside our first "official" scoring period ([#1](/1.md)) for the Founding Member Program, we will also be running a special one-off scoring period to compensate existing contributors to the project.
+
+`Intitial Founding Members` (who have already been contacted) will not be able to submit activity summaries as part of Scoring Period #0 as we have already accounted for their previous contributions.
+
+However, if you are not an `Initial Founding Member` but have already made contributions to the project, during the course of Scoring Period #1, simply submit a summary of all of your contributions to the project so far using the instructions [here](../SUBMISSION-GUIDELINES.md) and we will account for this activity with leaderboard points (in the same way as ongoing contributions).
+
+
+#### Processing Note
+
+_Once we have processed all of the summaries submitted in this period and inducted any additional founding members etc., a processing note will be added here with comments and some suggestions for the next scoring period._
+
+_Check back here on `DD/MM/YY` when we expect to have processed all of the qualifying activity summaries._
```

Comment:
I'm not sure we can promise a date here, as we don't know when people will submit.

Reply:
Let's instead say:
- will process within two weeks.
- must submit within 28/02/21 to guarantee grading?